### ISSUE TITLE
selectedValue does't change

### ISSUE THREAD

**Issue by kihot** [rob-balfre/svelte-select#109]
I use MultiSelect (isMulti={true}), but when the user selects a specific option, I need to remove the other selected elements. I use "on:select" and change selectedValue but Select is not updated.

**Comment by rob-balfre** [comment]
@kihot can you supply a REPL please?

**Comment by itay-grudev** [comment]
@rob-balfre I have the same problem. I've been fighting it for several days. And when I copy my code onto REPL it works fine.

**Comment by itay-grudev** [comment]
I asked for help on the Svelte Discord channel and `@TehShrike` suggested that the big difference  between my setup and what the REPL is doing – the REPL uses Rollup and doesn't actually import the built JS from `svelte-select` - it's pulling in the file from this entry point:

https://github.com/rob-balfre/svelte-select/blob/7d25aff598c45865a608380826183a8db98e523f/package.json#L6

and running it through its own compiler. The issue arises from the difference between the JS that `svelte-select` is shipping to npm (generated by svelte@3.6.7) and the JS spit out by my version of Svelte.

A quick workaround is to import:

```js
import Select from 'svelte-select/src/Select.svelte'
```
instead of

```js
import Select from 'svelte-select'
```
They further suggested that I should create PR and bump the svelte version up.

**Comment by subpx** [comment]
@itay-grudev are we still talking about the "selectedValue does't change" issue?

**Comment by itay-grudev** [comment]
@subpx Yes. @rob-balfre fixed it last night, though. Thanks for the quick reaction. It's fixed in `v3.4.0`
